### ISSUE TITLE
Rewrite - Heartbeats are still sent after client is destroyed causing process to segfault.

### ISSUE THREAD

**Issue by gabixdev** [FiniteReality/thermite#3]
```
[New Thread 0x7f8aa2396700 (LWP 14592)]
[New Thread 0x7f8aa3398700 (LWP 14593)]

Thread 40 "musicbot" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7f8aa93a4700 (LWP 14546)]
0x00007f8ae34dd514 in web::websockets::client::websocket_callback_client::send (this=0x7f8ac401b130, msg=...) at /usr/include/cpprest/ws_client.h:571
571	        return m_client->send(msg);
```

**Comment by FiniteReality** [comment]
Hey, could you try out the `issue/3` branch? I made a quick change which should fix it, but I'm not sure yet.

Could you also verify that calling `voice_client::stop()` also works, since it uses more or less the same code?

**Comment by FiniteReality** [comment]
Don't know why this wasn't closed, but should be repaired by 499baff
